Layouts/BlizzModern.lua

```diff
@@ -0,0 +1,263 @@
+local layoutName = "Blizz Modern"
+local layout = {}
+layout.name = "|cff00b4ffBlizz|r Modern by |cffFF7C0AMalithorn|r"
```

Comment:
Can we change this to just `Blizz Modern`? I will mention your contribution in README and on the Curseforge description. Is that okay?

Reply:
Or you want your name on the layout can just call it `Malithorn` without `Blizzard Modern`